cmd/admin/main.go

```diff
@@ -23,6 +23,9 @@ import (
 
 type Config struct {
 	Admin struct {
+		// Deprecated: DebugAddr is now a CLI flag only.
+		DebugAddr string
```

Comment:
I'd leave this in its original place. We only move deprecated feature flags to the top of that struct because we churn through them so quickly.